### ISSUE TITLE
[Question] FFMPEG Flags

### ISSUE THREAD

**Issue by soybeans-generator** [xbmc/inputstream.ffmpegdirect#313]
I have a stream url that every once in a while disconnect and to be able to play it in mpv, I used this flags:
```
--stream-lavf-o-append=reconnect_on_http_error=4xx,5xx --stream-lavf-o-append=reconnect_delay_max=30 --stream-lavf-o-append=reconnect_streamed=yes --stream-lavf-o-append=reconnect_on_network_error=yes
```
I am trying to have this kind of setup in kodi. Is it possible to set these flags in kodiprops? I have seen this repo with the reconnect flags here https://github.com/kodi-pvr/pvr.iptvsimple?tab=readme-ov-file#http-header-fields-supported-by-kodi but I don't know how to set it properly. Below configurations are my hail mary attempt at trying to make it work
```
#KODIPROP:inputstream=inputstream.ffmpegdirect
#KODIPROP:mimetype=video/mp2t
#KODIPROP:reconnect_on_http_error=4xx,5xx
#KODIPROP:reconnect_delay_max=30 
#KODIPROP:reconnect_streamed=yes
#KODIPROP:reconnect_on_network_error=yes
#EXTINF:-1, Channel 1
http://example.com
```
Can anyone provide a guide on how to do it properly or answer if it is not possible?

**Comment by phunkyfish** [comment]
No, you can't supply those parameters as Kodi props. Are they http values or arguments to pass to ffmpeg?
